[Bugfix] Fix Object Instantiation error

AlexBachmann opened the pull request:
Due to a missing "new" while instantiating the Vector2 object, the code breaks.

<!--
Thank you for contributing to Shopware! Please fill out this description template to help us to process your pull request.

Please make sure to fulfil our contribution guideline (https://docs.shopware.com/en/shopware-platform-dev-en/community/contribution-guideline?category=shopware-platform-dev-en/community).

Do your changes need to be mentioned in the documentation?
Add notes on your change right now in the documentation files in /src/Docs/Resources and add them to the pull request as well. 
-->

### 1. Why is this change necessary?


### 2. What does this change do, exactly?


### 3. Describe each step to reproduce the issue or behaviour.


### 4. Please link to the relevant issues (if any).


### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

janbuecker commented:
Thanks @t2oh4e 

With Shopware 5.2, we've refactored a lot code and also removed some parts that are relevant for this PR. As we cannot easily merge this now, please create a new PR if the problem still exists.

AlexBachmann committed:
[Bugfix] Fix Object Instantiation error

Due to a missing "new" while instantiating the Vector2 object, the code breaks.

mitelg commented:
thanks for your contribution @AlexBachmann :+1: :tada: :blue_heart:
